python/ray/tests/test_dynres.py

```diff
@@ -569,3 +569,25 @@ def delete_res(resource_name, res_client_id):
             all_resources_created.append(str(i) in resources)
         success = all(all_resources_created)
     assert success
+
+
+def test_release_cpus_when_actor_creation_task_blocking(shutdown_only):
+    ray.init(num_cpus=2)
+
+    @ray.remote(num_cpus=1)
```

Comment:
```suggestion
    @ray.remote
```

This is the default.

Reply:
Thanks! But I specify it intentionally to indicates this task requires 1 CPU explicitly. I think this is helpful for other readers.